flytekit/clis/sdk_in_container/serialize.py

```diff
@@ -320,6 +321,15 @@ def fast_workflows(ctx, folder=None):
     if folder:
         click.echo(f"Writing output to {folder}")
 
+    source_dir = ctx.obj[CTX_LOCAL_SRC_ROOT]
```

Comment:
moved this up so the green success statement at the end of serialize prints last